squid-mixin/links.libsonnet

```diff
@@ -0,0 +1,13 @@
+local g = import './g.libsonnet';
+local commonlib = import 'common-lib/common/main.libsonnet';
+
+{
+  new(this):
+    {
+      overview: {
```

Comment:
This is the incorrect way of structuring the dashboard link.

Follow the pattern here: https://github.com/grafana/jsonnet-libs/blob/master/apache-activemq-mixin/links.libsonnet